terminus/generators/opendrive_id_mapper.py

```diff
@@ -0,0 +1,11 @@
+from city_visitor import CityVisitor
+
+
+class OpenDriveIdMapper(CityVisitor):
```

Comment:
Ok, makes sense. Can you add a class comment stating that so we know this class will get further development?